src/lib/SetPreview.svelte

```diff
@@ -57,7 +57,7 @@
 		color: inherit;
 	}
 	.set-preview {
-		padding: 1rem;
+		padding: var(--set-preview-padding, 0.5em);
```

Comment:
Is it intentional that the fallback isn't the same as what's set in `app.css`?

Reply:
yes? I think in this case it looks a bit better with a bit more space, but with no styles it looks better a bit tighter